www/src/components/repository/RepositoryDeployments.js

```diff
@@ -111,7 +111,6 @@ function RepositoryDeployments() {
       />
       <Div
         fill
-        marginTop="medium"
```

Comment:
As in https://github.com/pluralsh/plural/pull/399/commits/fd8af7eb8ff0b413e89abaa82cae3ef8f1271ede#r940113280.